Scripts/FirebaseInitializer.cs

```diff
@@ -74,9 +74,9 @@ public static class FirebaseInitializer {
         }
         if (!fetching) {
 #if UNITY_EDITOR
-          var settings = FirebaseRemoteConfig.Settings;
+          var settings = FirebaseRemoteConfigDeprecated.Settings;
```

Comment:
`FirebaseRemoteConfig.DefaultInstance.ConfigSettings`?
Similar to those in below